deployments/data-hub/automations/data-hub-core-dags-stable-policy.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: image.toolkit.fluxcd.io/v1beta2
```

Comment:
Just a thought: We could also combine definitions into a single file like we done for the deployment, e.g. policy and source, or all policies in one file.

There will be one more repo (for the eJP XML pipeline).